pkg/console/config/types.go

```diff
@@ -0,0 +1,56 @@
+package consoleserver
```

Comment:
Not sure if its the right name for this pkg ...

Reply:
should be `config`